### ISSUE TITLE
Add a mechanism to sync the mempool

### ISSUE THREAD

**Issue by styppo** [nimiq/core-rs-albatross#2625]

**Comment by sisou** [comment]
Is the mempool not synced between validators?

**Comment by hrxi** [comment]
No, they're just accidentally synced if they see the same transactions.

**Comment by sisou** [comment]
As mentioned in https://github.com/nimiq/core-rs-albatross/pull/2218#discussion_r1637657311, should full nodes and light nodes also be able to see the mempool?
